lib/client/signalflow/signalflow_client.js

```diff
@@ -74,6 +74,44 @@ function SignalflowClient(apiToken, options) {
     };
   }
 
+  function liveTailRequest() {
```

Comment:
This function probably would be taking an `opts` argument that would contain the user's query.